seacrowd/utils/schemas/image_text.py

```diff
@@ -12,7 +12,18 @@ def features(label_names = ["Yes", "No"]):
             "metadata": {
                 "context": datasets.Value("string"),
                 "labels": datasets.Sequence(datasets.ClassLabel(names=label_names)),
-            }
+                "annotations": datasets.Sequence({
+                    "x1": datasets.Value("int32"),
+                    "y1": datasets.Value("int32"),
+                    "x2": datasets.Value("int32"),
+                    "y2": datasets.Value("int32"),
+                    "x3": datasets.Value("int32"),
+                    "y3": datasets.Value("int32"),
+                    "x4": datasets.Value("int32"),
+                    "y4": datasets.Value("int32"),
+                    "transcript": datasets.Value("string"),
+                })
+            },
```

Comment:
Initially I thought abt that too, but this implementation has been discussed in discord discussion [thread](https://discord.com/channels/1164061531227557920/1227907459897757747). Wdyt guys? @akhdanfadh @holylovenia

Reply:
@patrickamadeus noted.
I do agree with the thread, i.e. for image & text dataset we should use this format.
However, does not this change (i.e. making the schemas having bounding box annotations) necessitate the change of other datasets without the annotations?
cc: @akhdanfadh @holylovenia

Reply:
@patrickamadeus The `annotations` is supposed to be a key in the `metadata` according to our discussion. It was some time ago though, so it makes sense if we forget about this little detail.

<img width="487" alt="image" src="https://github.com/SEACrowd/seacrowd-datahub/assets/25114282/04c1649e-1ce6-4101-bf8b-086164685606">

Reply:
Can I ask something, since we are not supporting the task of Scene Text Recognition, why do we _need_ the BB in the metadata?

As this is formulated as an OCR task, _not STR or text localisation_, I would suggest it is OK not to _force_ BB labels in the SEACrowd schema. For 2 reasons:

1. The BB is accessible from the source schema anyway.
2. There is no need to change the `utils/schemas/image_text.py`, but put the BB labels in this particular dataloader only, i.e.: only in the `sea_datasets/vintext/vintext.py`.

Reply:
@fhudi `metadata` already exists in the current [`imtext` schema](https://github.com/SEACrowd/seacrowd-datahub/blob/master/seacrowd/utils/schemas/image_text.py). Does adding the `annotations` inside the `metadata` trigger an error for the unit test?

Reply:
@fhudi The purpose is precisely to store all the additional information such as bbox. That's intentional. It's good that it doesn't trigger any error, let's keep the `annotations` inside the `metadata` then.

cc: @patrickamadeus

Reply:
in that case, can we put some comments on the schema file indicating some keys may not be available depending on the dataloader? 🙏

Reply:
@fhudi Done.

Reply:
Doing it rn!